.agent-os/watch/watch-specs.sh

```
@@ -0,0 +1,94 @@
+    fi
+    
+    # Create implementation directory structure
+    mkdir -p "$impl_path/files"
+    mkdir -p "$impl_path/tests"
+    mkdir -p "$review_path"
+    
+    # Update status to processing
+    echo '{"status": "processing", "started_at": "'$(date -Iseconds)'"}' > "$spec_path/status.json"
+    
+    # Create implementation request file for Claude Code
+    cat > "$impl_path/request.md" << EOF
```

Shellcheck (0.10.0)

[error] Couldn't parse this here document. Fix to allow more checks.

(SC1073)